### PR TITLE
8.0 fix some errors in Country states [Régions]

### PR DESCRIPTION
**Pull request by DayssamAgora**

### CODE REVIEW

**Comment by DayssamAgora** [comment]
@max3903 
It's OK ?

File: .settings/org.eclipse.core.resources.prefs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+encoding//l10n_ma_state/README.rst=UTF-8
+encoding//l10n_ma_state/__init__.py=utf-8
+encoding//l10n_ma_state/__openerp__.py=utf-8
```
Comment (max3903) [inline .settings/org.eclipse.core.resources.prefs]: Add this file to the .gitignore and remove it

**Comment by max3903** [comment]
You need to rebase your branch:

```
git remote add oca git://github.com:OCA/l10n-morocco
git fetch oca
git rebase -i oca/8.0
```

https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#how-do-i-rebase

**Comment by DayssamAgora** [comment]
@max3903 : the rebase is done

**Comment by max3903** [comment]
@DayssamAgora I don't think it is. You should have the following commits in your history:
https://github.com/OCA/l10n-morocco/commits/8.0

Let me do it.

**Comment by max3903** [comment]
@DayssamAgora You still have few errors:
https://travis-ci.org/OCA/l10n-morocco/jobs/100668817#L373

**Comment by max3903** [comment]
@DayssamAgora and there are still references to the author of the original module:
https://github.com/AgoraDev/l10n-morocco/blob/8.0-state/l10n_ma_state/__init__.py#L6
https://github.com/AgoraDev/l10n-morocco/blob/8.0-state/l10n_ma_state/__openerp__.py#L6

**Comment by max3903** [comment]
@DayssamAgora State code is limited to 3 characters so all your states are imported with the code `MA-`

contacts - odoo 2016-01-07 10-01-45

**Comment by max3903** [comment]
:+1:

**Comment by leemannd** [comment]
:+1:

**Pull request merged**